packages/vscode-extension/src/commonlib/telemetry.ts

```diff
@@ -19,19 +31,53 @@ import { Correlator } from "@microsoft/teamsfx-core";
 export class VSCodeTelemetryReporter extends vscode.Disposable implements TelemetryReporter {
   private readonly reporter: Reporter;
   private readonly extVersion: string;
+  private readonly logger: Logger;
+  private readonly testFeatureFlag: boolean;
 
   private sharedProperties: { [key: string]: string } = {};
 
   constructor(key: string, extensionVersion: string, extensionId: string) {
     super(async () => await this.reporter.dispose());
     this.reporter = new Reporter(extensionId, extensionVersion, key, true);
     this.extVersion = getPackageVersion(extensionVersion);
+    const logFile = path.join(os.homedir(), `.${ConfigFolderName}`, TelemetryTestLoggerFile);
+    configure({
+      appenders: { everything: { type: "file", filename: logFile } },
+      categories: { default: { appenders: ["everything"], level: "debug" } },
+    });
+    this.logger = getLogger("TelemTest");
```

Comment:
Is the log file created here or on the first log generated?

Reply:
created at 'getLogger' call.

Reply:
oh, was wrong, it's at 'configure' call.

Reply:
So how about move `configure` and `getLogger` in if condition? For normal user we don't want to generate the test log file.

```
if (TelemetryTest) {
   configure...
   getLogger...
}
```